code/modules/projectiles/projectile.dm

```diff
@@ -102,6 +105,11 @@ var/list/impact_master = list()
 	var/fire_sound = 'sound/weapons/Gunshot.ogg' //sound that plays when the projectile is fired
 	var/rotate = 1 //whether the projectile is rotated based on angle or not
 
+/obj/item/projectile/New()
+	..()
+	initial_pixel_x = pixel_x
```

Comment:
I swear some maintainer told me that it was bad some time ago. It might not have been you then. But in that case what's your opinion on using initial() twice at every step?